service/src/main/java/com/bankbazaar/kafka/service/consumer/KafkaConsumer.java

```diff
@@ -13,7 +13,7 @@
 @Slf4j
 @Service
 public class KafkaConsumer {
-    @KafkaListener(topics = "create_CSV")
+    @KafkaListener(topics = "create_CSV", groupId = "Partner_Integration_Kafka_Group")
```

Comment:
GroupId should be something related to your project. Also keep all configs like this in application.properties